apps/designer/app/shared/db/project.server.ts

```diff
@@ -6,41 +6,55 @@ import {
   prisma,
   Prisma,
   Project as BaseProject,
+  Asset,
 } from "@webstudio-is/prisma-client";
 import * as db from ".";
+import { getAssetPath } from "@webstudio-is/asset-uploader/src/helpers/get-asset-path";
 
 const TreeHistorySchema = z.array(z.string());
 
 export type Project = Omit<BaseProject, "prodTreeIdHistory"> & {
   prodTreeIdHistory: z.infer<typeof TreeHistorySchema>;
 };
 
-const parseProject = (project: BaseProject | null): Project | null => {
+const parseProject = (
+  project: BaseProject | null
+): Project | Omit<BaseProject, "assets"> | null => {
```

Comment:
This return type looks a little bit fishy - I would expect a unified type to come up from this parsing function (so probably smth like `Project | null`).